contents/node/nodejs.md

```diff
@@ -62,31 +62,43 @@ Server running at http://127.0.0.1:3000/
 ```
 To test the server, open a browser tab and navigate to http://localhost:3000/. You should see 'Hello world'.
 
+</box>
+
 Read on to find out about how to incorporate external dependencies, manage them and use Node's module system to organize your code.
 
 ### Benefit 2: Avoid Synchronization Problems and Overheads
 
-Node is designed to be **event-driven**. When an event happens e.g. an I/O event is complete, the event handler/callback function will be enqueued in a queue where it will be subsequently scheduled to run by the [event loop](https://medium.com/the-node-js-collection/what-you-should-know-to-really-understand-the-node-js-event-loop-and-its-metrics-c4907b19da4c). The main event loop (where Javascript code is executed) is single-threaded.
+<figure>
+<img src="eventloop.png" width="74%" class="mx-auto mt-3 mb-2 d-block rounded" />
+<figcaption class="text-center"><small>
```

Comment:
Consider mentioning or making reference to this figure in the following paragraph so it is not just "there" without anything talking about it.